pkgs/development/python-modules/certbot-dns-rfc2136/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, acme
+, certbot
+, dnspython
+, isPy3k
+, pytest
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  inherit (certbot) src version;
+
+  pname = "certbot-dns-rfc2136";
+
+  propagatedBuildInputs = [
+    acme
+    certbot
+    dnspython
+  ];
+
+  checkInputs = [
+    pytest
+    pytestCheckHook
+  ];
+
+  disabled = !isPy3k;
+
+  pytestFlagsArray = [ "-o cache_dir=/tmp" ];
```

```suggestion
  pytestFlagsArray = [ "-o cache_dir=$(mktemp -d)" ];
```

Will probably make this slightly more portable.

---

fixed.